src/operators/products.c

```diff
@@ -180,25 +353,59 @@ static igraph_error_t tensor_product(igraph_t *res,
  * index in the product graph using <code>index = u |V2| + v</code>.
  *
  * </para><para>
+ * All implemented graph products are associative, but not all are commutative.
+ *
+ * </para><para>
  * The supported graph product types are detailed below. The notation
  * <code>u ~ v</code>
  * is used to indicate that vertices \c u and \c v are adjacent, i.e. there is
  * a connection from \c u to \c v.
  * \clist
- *    \cli IGRAPH_PRODUCT_CARTESIAN
- *    Computes the Cartesian product of two graphs. In the product graph,
- *    there is a connection from <code>(u1, v1)</code> to <code>(u2, v2)</code>
- *    if and only if
- *    <code>u1 = u2</code> and <code>v1 ~ v2</code> or
- *    <code>u1 ~ u2</code> and <code>v1 = v2</code>.
- *    Thus, the number of edges in the product is
- *    <code>|V1| |E2| + |V2| |E1|</code>.
+ *     \cli IGRAPH_PRODUCT_CARTESIAN
+ *     Computes the Cartesian product of two graphs. In the product graph,
+ *     there is a connection from <code>(u1, v1)</code> to <code>(u2, v2)</code>
+ *     if and only if
+ *     <code>u1 = u2</code> and <code>v1 ~ v2</code> or
+ *     <code>u1 ~ u2</code> and <code>v1 = v2</code>.
+ *     Thus, the number of edges in the product is
+ *     <code>|V1| |E2| + |V2| |E1|</code>.
  *
  *     </para><para>
  *     Time Complexity: O(|V1| |V2| + |V1| |E2| + |V2| |E1|)
```

Comment:
`Complexity` should be lowercase to make it consistent with other parts of the documentation. This applies to all occurrences of `Time Complexity` within this PR.